web/src/utils/uuidToURL.ts

```diff
@@ -0,0 +1,20 @@
+export const uuidToURL = (uuid: string, objectName: string) => {
+  // TODO: needs proper implementation
+  if (!uuid || !objectName) return "";
+  const storageHost = window.REEARTH_CONFIG?.storageHost ?? "http://localhost:8080";
+  const assetBase = window.REEARTH_CONFIG?.assetBase ?? "assets";
+  const path = getPathFromUUID(uuid, objectName);
+  if (!path) return "";
+  return joinPath(storageHost, assetBase, path);
+};
+
+export const getPathFromUUID = (uuid: string, objectName: string) => {
+  if (!uuid || !objectName) return "";
```

Comment:
Same as above. I dont htink this is necessary

Reply:
Done!